device.go

```diff
@@ -253,7 +253,12 @@ func GetHostPath(devInfo DeviceInfo) (string, error) {
 
 	content, err := ini.Load(sysDevPath)
 	if err != nil {
-		return "", err
+		// Some devices(eg. /dev/fuse, /dev/cuse) do not always implement sysfs interface under /sys/dev
+		// These devices are passed by default by docker.
+		//
+		// Simply return the path passed in the oci.spec file, this does mean that no device renames are
```

Comment:
I would prefer to avoid mentioning `oci.spec` since this is OCI specific. I think we can just say that we instead provide the path that has been passed through the device configuration.